backend/src/main/java/de/neuefische/backend/service/BagPlacesService.java

```diff
@@ -0,0 +1,43 @@
+package de.neuefische.backend.service;
+
+
+import de.neuefische.backend.BackendApplication;
+import de.neuefische.backend.model.BagPlaces;
+import de.neuefische.backend.model.BagPlacesDTO;
+import de.neuefische.backend.repo.BagPlacesRepository;
+import org.apache.juli.logging.Log;
+import org.apache.juli.logging.LogFactory;
+import org.springframework.stereotype.Service;
+import org.springframework.validation.annotation.Validated;
+
+import java.util.List;
+
+@Service
+public class BagPlacesService {
+
+    private static final Log LOG = LogFactory.getLog(BackendApplication.class);
+    private final BagPlacesRepository bagPlacesRepository;
+
+    public BagPlacesService(BagPlacesRepository bagPlacesRepository) {
+        this.bagPlacesRepository = bagPlacesRepository;
+    }
+
+    public List<BagPlaces> getAll(){
+        return bagPlacesRepository.findAll();
+    }
+
+    public void createBagPlace(@Validated BagPlacesDTO data){
+        final BagPlaces bagPlace = BagPlaces.newBagPlace(data.getId(), data.getUsername(), data.getCreationDate(), data.getLat(), data.getLng());
+        try {
+            bagPlacesRepository.insert(bagPlace);
+            LOG.info("New BagPlace created...");
+            LOG.info("ID: { " + bagPlace.getId() + " }");
+            LOG.info("Creator: { " + bagPlace.getUsername() + " }");
```

Comment:
Better don't log user information

Reply:
yeah, you're right, just needed it for testing. will be deleted later :)